src/Native/ObjWriter/llvmCap/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required(VERSION 3.6)
+project(ObjWriter)
+include(ExternalProject)
+
+set(OBJWRITER_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../)
+set(OBJWRITER_LLVM_POINT tools/ObjWriter)
+
+if(NOT OBJWRITER_TARGET_ARCH)
+  if(CLR_CMAKE_TARGET_ARCH MATCHES "^(x64|x86)$")
+    if(NOT $ENV{__CrossBuild})
+      set(OBJWRITER_TARGET_ARCH "X86")
+    else()
+      # If we really want a cross version for x86/x64 -> arm32/arm64
+      # I think in the opposite direction it's not necessary for anyone ;-)
+      set(OBJWRITER_TARGET_ARCH "X86::ARM")
+      set(USE_ARM_TARGET_TRIPLE 1)
+    endif()
+  elseif(CLR_CMAKE_TARGET_ARCH MATCHES "^(arm|armel)$")
+    set(OBJWRITER_TARGET_ARCH "ARM")
+    set(USE_ARM_TARGET_TRIPLE 1)
+  elseif(CLR_CMAKE_TARGET_ARCH STREQUAL "arm64")
+    set(OBJWRITER_TARGET_ARCH "AArch64")
+  else()
+    clr_unknown_arch()
+  endif()
+endif()
+
+if(NOT OBJWRITER_BUILD_TYPE)
+  set(OBJWRITER_BUILD_TYPE "Release")
+endif()
+
+get_target_property(CORERT_NATIVE_COMPILE_OPTIONS Runtime COMPILE_OPTIONS)
+
+# Make sure to remove debug flags from general build flags in Release case
+if(OBJWRITER_BUILD_TYPE STREQUAL "Release")
+  list(REMOVE_ITEM CORERT_NATIVE_COMPILE_OPTIONS "-g")
```

Comment:
Our convention has been to build with -g in Release, and then archive and strip the symbols on the binary.

Reply:
Ok, just llvm picks up "-g" and binaries become very large. Of course, we can archive and strip it manually.